test/augmentation/test_augmentation_mix.py

```diff
@@ -387,3 +387,113 @@ def test_p(self, p, same_on_batch, device, dtype):
         input = torch.randn((12, 3, 256, 256), device=device, dtype=dtype)
 
         f(input)
+
+
+class TestRandomTransplantation:
```

Comment:
Kornia also has a `BaseTester`, which provides `self.assert_close` to be used instead of `torch.allclose`

[Checkout our testing guide](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.md#coding-standards)